src/net/sourceforge/plantuml/classdiagram/command/CommandHideShowByGender.java

```diff
@@ -163,6 +163,10 @@ private CommandExecutionResult executeClassDiagram(AbstractClassOrObjectDiagram
 			gender = EntityGenderUtils.byEntityType(LeafType.ABSTRACT_CLASS);
 		} else if (arg1.equalsIgnoreCase("annotation")) {
 			gender = EntityGenderUtils.byEntityType(LeafType.ANNOTATION);
+		} else if (arg1.equalsIgnoreCase("protocol")) {
```

Comment:
Hello @bobjacobsen, and all,

Why the `executeDescriptionDiagram` function _(see line 134)_:

https://github.com/bobjacobsen/plantuml/blob/2b337ac940996904fa2f41d3135728b63986fcf0/src/net/sourceforge/plantuml/classdiagram/command/CommandHideShowByGender.java#L133-L134

does not have the same lines:
```java
		} else if (arg1.equalsIgnoreCase("protocol")) {
			gender = EntityGenderUtils.byEntityType(LeafType.PROTOCOL);
		} else if (arg1.equalsIgnoreCase("struct")) {
			gender = EntityGenderUtils.byEntityType(LeafType.STRUCT);
```